rpc/server.go

```diff
@@ -366,6 +366,11 @@ func NewServer(logger golog.Logger, opts ...ServerOption) (Server, error) {
 		server.exemptMethods["/proto.rpc.v1.AuthService/Authenticate"] = true
 	}
 
+	if sOpts.allowUnauthenticatedHealthCheck {
+		server.exemptMethods["/viam.health.v1.Health/Check"] = true
```

Comment:
why not use the grpc health service https://github.com/grpc/grpc/blob/master/doc/health-checking.md? also, could we run a grpc server on a separate port for this so we dont expose this to the public for app.viam

Reply:
We can use this: https://pkg.go.dev/google.golang.org/grpc/health to implement the check, but i wanted to added a check for the mongo DB too as we have had deployments cause the server to go down in the past.

Reply:
so what I'm saying is we just implement that service instead of using the default code.

Reply:
and having a health check port option